component_contribution/core/compound.py

```diff
@@ -4,12 +4,14 @@
 import openbabel
 import numpy as np
 import requests
-from component_contribution.exceptions import OpenBabelError
 
 from scipy.misc import logsumexp
 
 import bioservices
 
+from cobra.core.metabolite import Metabolite
```

Comment:
For an external package it's better to import from the top level, i.e., `from cobra import Metabolite`.